### PR TITLE
[threejs] add Intersect Type

### PR DESCRIPTION
**Pull request by FishOrBear**

### CODE REVIEW

File: types/three/three-core.d.ts [hunk 1]
```diff
@@ -5171,7 +5181,7 @@ export class Mesh extends Object3D {
     setDrawMode(drawMode: TrianglesDrawModes): void;
     updateMorphTargets(): void;
     getMorphTargetIndexByName(name: string): number;
-    raycast(raycaster: Raycaster, intersects: any): void;
+    raycast(raycaster: Raycaster, intersects: Intersect[]): Intersect[];
```
Comment (dhritzkiv) [inline types/three/three-core.d.ts]: Does `raycast` indeed return an array of intersects? I was looking through the [sourcecode](https://github.com/mrdoob/three.js/blob/5e7a7b83e1e182e662fd7b60a2674f85c9a57b35/src/objects/Mesh.js#L225) and couldn't find a return statement that returned Intersects. Only void.

I believe the THREE documentation for the `raycast` method is wrong (where is suggests it returns an array)

**Review by dhritzkiv — APPROVED** [review]

File: types/three/three-core.d.ts [hunk 2]
```diff
@@ -5158,6 +5158,16 @@ export class LineSegments extends Line {
     );
 }
 
+interface Intersect
+{
```
Comment (DanielRosenwasser) [inline types/three/three-core.d.ts]: Can you keep style consistent in this file?

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

**Comment by DanielRosenwasser** [comment]
Just as a heads up, your commits don't seem to be associated with your GitHub account. While this isn't technically a problem, you might care if you want more appropriate attribution. You can either make sure your GitHub account is associated with the email address you're using for your commits, or rebase and amend your commits to fix the author name and email.

If you don't care about any of this stuff, I can always just pull in the changes.

**Comment by FishOrBear** [comment]
@DanielRosenwasser    Thanks for your reminder, I have updated my email.

**Comment by FishOrBear** [comment]
@DanielRosenwasser  Wait, I learn how to modify.

**Pull request closed without merging**